iot-hub/Samples/service/CleanupDevicesSample/CleanupDevicesSample.cs

```diff
@@ -65,8 +69,29 @@ private async Task CleanupDevices()
                     excludeKeysInExport: true,
                     storageAuthenticationType: StorageAuthenticationType.KeyBased);
 
-            JobProperties exportAllDevicesJob = await _registryManager.ExportDevicesAsync(exportAllDevicesProperties);
+            JobProperties exportAllDevicesJob = null;

```

Comment:
Adding retries on jobs to protect against any currently running job. We could change the total wait time if necessary after observing for a while.